ml-agents/mlagents/trainers/stats.py

```diff
@@ -23,6 +24,10 @@ def empty() -> "StatsSummary":
         return StatsSummary(0.0, 0.0, 0)
 
 
+class StatsPropertyType(Enum):
+    hyperparameters = "hyperparameters"
```

Comment:
```suggestion
    HYPERPARAMETERS = "hyperparameters"
```
I think generally python enums use SCREAMING_SNAKE_CASE.